### PR TITLE
1846: hotseat draft displays drafted companies and adjusted cash/value

### LINKED ISSUE #727: [1846] Private Draft doesn't show a player the companies they've drafted
During the private draft (in a hotseat game), nothing shows the current player how much money they have already allocated for the privates they've already chosen, or what privates they've already chosen. I realize that's hidden information until the end of the draft, but a player should be able to see that information when they click "show".

### PR DESCRIPTION
**Pull request by jaysachs**
Fixes #727.

### CODE REVIEW

File: assets/app/view/game_page.rb [hunk 1]
```diff
@@ -231,7 +231,7 @@ def render_action
       when Engine::Round::Operating
         h(Game::Round::Operating, game: @game)
       when Engine::Round::G1846::Draft
-        h(Game::Round::Draft, game: @game)
+        h(Game::Round::Draft, game: @game, round: @round)
```
Comment (tobymao) [inline assets/app/view/game_page.rb:234]: this is unnecessary, you can get @game.round

File: assets/app/view/game/round/draft.rb [hunk 2]
```diff
@@ -85,9 +86,31 @@ def render_input
         def render_player
           return nil if @hidden
 
-          h(:div, [
-            h(Player, player: @player, game: @game),
-          ])
+          draft_dist = @round.steps[-1]
+          drafted = draft_dist.choices[@player]
+
+          player = @player.clone
```
Comment (tobymao) [inline assets/app/view/game/round/draft.rb:92]: i'm not a big fan of this approach, it seems a bit hacky, can we instead do a simpler view?

Comment (tobymao) [reply assets/app/view/game/round/draft.rb:92]: just listing the companies name that have been drafted and the remaining money so we don't have to clone and mock objects?

**Comment by jenf** [comment]
solution based on suggest approach done as #1307

**Pull request closed without merging**